go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c // indirect
 	github.com/xdg/stringprep v1.0.0 // indirect
 	github.com/yuin/gopher-lua v0.0.0-20181031023651-12c4817b42c5 // indirect
+	go.mongodb.org/mongo-driver v1.0.2
```

Comment:
do we need to remove "github.com/mongodb/mongo-go-driver"?

Reply:
MongoDB go driver >=v.1.0.0 imported from go.mongodb.org/mongo-driver  [https://github.com/mongodb/mongo-go-driver](https://github.com/mongodb/mongo-go-driver)
when i was building the code, it keeps pulling from "github.com/mongodb/mongo-go-driver" v.0.3 even though i have manually added and installed the newest driver. Therefore i replaced it with go.mongodb.org/mongo-driver.